xm-commons-idp/src/main/java/com/icthh/xm/commons/domainevent/idp/IdpConfigUtils.java

```diff
@@ -1,9 +1,9 @@
-package com.icthh.xm.commons.domain.idp;
```

Comment:
@EvgeniyMurenko why do you touch IDP feature related classes?

Reply:
made a transfer using a patch from the "Outbox" branch.
rollback